tests/subsys/shell/modules/kernel_service/src/main.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
```

Comment:
dislike this type of  build_only "test", especially if we have this already covered somewhere else, i.e. samples/subsys/shell/shell_module. Either add a new prj.conf there with additional options or make this a proper test please using pytest

Reply:
done

Reply:
Had some trouble getting the test right so I've dropped them, they will be done later in a separate PR